cfgen/cfgen

```diff
@@ -1410,11 +1412,11 @@ class ConfPool(ToDhall):
                 # indivisible pieces of data, the whole CAS record is
                 # always stored on one node.
                 data_units = 1
-                parity_units = len(cluster_encls(m0conf)) - 1
+                parity_units = len(cluster_svcs(m0conf, SvcT.M0_CST_CAS)) - 1
```

Comment:
I missed this somehow, good point by Madhav, idea to use number of enclosures is to have redundancy factor for metadata with respect to nodes. As 1 node can also run multiple ioservices, it this case with replication more that the nodes will lead to to much redundancy. We want to restrict replication factor with respect to the number of node failures. In worst case it is okay to have it equal to number of nodes instead of number of ioservices.

Reply:
Good point. But we need to take into account server nodes, not just any nodes (which can be client-only ones).

Reply:
Updated the patch, please check.